[2025-01-03T07:28:51.480+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: insert_data.ft_posting_f scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-03T07:28:51.496+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: insert_data.ft_posting_f scheduled__2024-12-31T00:00:00+00:00 [queued]>
[2025-01-03T07:28:51.496+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 3
[2025-01-03T07:28:51.727+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): ft_posting_f> on 2024-12-31 00:00:00+00:00
[2025-01-03T07:28:51.735+0000] {standard_task_runner.py:60} INFO - Started process 90 to run task
[2025-01-03T07:28:51.740+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'insert_data', 'ft_posting_f', 'scheduled__2024-12-31T00:00:00+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp576lwv6d']
[2025-01-03T07:28:51.744+0000] {standard_task_runner.py:88} INFO - Job 203: Subtask ft_posting_f
[2025-01-03T07:28:51.804+0000] {task_command.py:423} INFO - Running <TaskInstance: insert_data.ft_posting_f scheduled__2024-12-31T00:00:00+00:00 [running]> on host 403e7aadd7ee
[2025-01-03T07:28:51.906+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='agushchin' AIRFLOW_CTX_DAG_ID='insert_data' AIRFLOW_CTX_TASK_ID='ft_posting_f' AIRFLOW_CTX_EXECUTION_DATE='2024-12-31T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-12-31T00:00:00+00:00'
[2025-01-03T07:28:51.952+0000] {base.py:83} INFO - Using connection ID 'postgres-db' for task execution.
[2025-01-03T07:28:52.730+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-01-03T07:28:52.739+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=insert_data, task_id=ft_posting_f, execution_date=20241231T000000, start_date=20250103T072851, end_date=20250103T072852
[2025-01-03T07:28:52.796+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-01-03T07:28:52.824+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
